akka-docs/src/test/scala/docs/stream/operators/sourceorflow/MergeLatest.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package docs.stream.operators.sourceorflow
+import akka.actor.ActorSystem
+import akka.stream.scaladsl.Source
+
+object MergeLatest extends App {
+  implicit val system = ActorSystem()
+
+  //#mergeLatest
+  val prices = Source(List(100, 101, 99, 103))
+  val quantity = Source(List(1, 3, 4, 2))
+
+  prices.mergeLatest(quantity).map(priceAndQuantity => priceAndQuantity(0) * priceAndQuantity(1)).runForeach(println)
```

Comment:
Could perhaps pattern match to give nice names? Is it a List or what?

`{ case price :: quantity :: Nil => price * quantity }`